apps/wakunode2/external_config.nim

```diff
@@ -150,11 +150,6 @@ type
       defaultValue: 0
       name: "rln-relay-membership-index" }: uint
 
-    rlnRelayMembershipGroupIndex* {.
-      desc: "the index of credentials to use, within a specific rln membership set",
-      defaultValue: 0
-      name: "rln-relay-membership-group-index" }: uint
-
```

Comment:
We cannot directly remove this parameter. Instead, we should inform that this is going to be deprecated two `nwaku` versions later than the current one.